.ci/onnx/jenkins/utils/utils.groovy

```diff
@@ -86,10 +86,12 @@ def CreateStage(String stageName, Closure method, configurationMaps) {
         try {
             def prepareEnvMap = GenerateMap(genericBodyMethod, configurationMaps)
             parallel prepareEnvMap
-        } catch(hudson.AbortException e) {
-            AbortedException(e)
         } catch(e) {
-            Exception(e)
+            if ("$e".contains("143")) {
```

Comment:
Please add a comment explaning the reson for using "143" here.

Reply:
Done